estimate.src.ipynb

```diff
@@ -707,7 +696,9 @@
    },
    "outputs": [],
    "source": [
-    "residual_df = -eval_df.subtract(eval_df.truth, axis=0).dropna().drop('truth', axis=1)"
```

Comment:
this df no longer had 'truth' in it due to changes in the plotting in a previous cell